tls/s2n_config.c

```diff
@@ -68,19 +69,27 @@ static int s2n_config_setup_default(struct s2n_config *config)
 {
     GUARD(s2n_config_set_cipher_preferences(config, "default"));
     GUARD(s2n_config_set_signature_preferences(config, "default"));
+    GUARD(s2n_config_set_ecc_preferences(config, "default"));
     return S2N_SUCCESS;
 }
 
 static int s2n_config_setup_tls13(struct s2n_config *config)
 {
     GUARD(s2n_config_set_cipher_preferences(config, "default_tls13"));
     GUARD(s2n_config_set_signature_preferences(config, "default_tls13"));
+    if (s2n_is_modern_ec_supported()) {
```

Comment:
Is this necessary? What happens if a customer uses "default_tls13" but modern ec isn't supported?

Reply:
This was necessary as I got rid of the  `#if` derivities here: https://github.com/awslabs/s2n/pull/1693/files#diff-3a0feac123c5470ac8401130f680b3dcR29.  If a customer uses `default_tls13` and `modern ec` is not supported we would want to fall back to a list which contains only `secp256r1 and secp384r1`.